kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/bluetooth/le/beacon/BluetoothAdvertisementData.java

```diff
@@ -0,0 +1,93 @@
+/*******************************************************************************
+ * Copyright (c) 2017 Eurotech and/or its affiliates
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Eurotech
+ *******************************************************************************/
+package org.eclipse.kura.bluetooth.le.beacon;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.osgi.annotation.versioning.ProviderType;
+
+/**
+ * @noextend This class is not intended to be subclassed by clients.
+ */
+@ProviderType
+public class BluetoothAdvertisementData {
+
+    private byte[] rawData;
+    private byte packetType;
```

Comment:
In this case, there are a lot of packet and event codes and most of them aren't very useful for our applications. I prefer to leave the code unchanged for now.